ambari-agent/src/main/python/ambari_agent/AmbariConfig.py

```diff
@@ -66,7 +67,7 @@
 passphrase_env_var_name=AMBARI_PASSPHRASE
 
 [heartbeat]
-state_interval = 6
+state_interval = 1
```

Comment:
Just curious - is this minutes or seconds?  1 second may be too fast.

Reply:
does stomp even heart-beating? i thinked they use WebSockets....if i'm not missing something

Reply:
If that's the case then the interval should not even be declared

Reply:
This part of code is used only in unit-tests (sadly we have this kind of hack for years in our code). Please check ambari-agent.ini for real default configs values.

It is 
state_interval_seconds=60